src/lib/profile/experience/validate-academic-experience.ts

```diff
@@ -0,0 +1,41 @@
+import { object, string, boolean, date, ref } from 'yup'
+
+const schema = object({
+  organizationName: string().required('Campo requerido'),
+  jobTitle: string().required('Campo requerido'),
+  description: string().required('Campo requerido'),
+  entryDate: date().nullable().default(null).required('Campo requerido'),
```

Comment:
```suggestion
  entryDate: date().required('Campo requerido'),
```
required anula el efecto de nullable

Reply:
cierto, arreglao